doc/config.md

```diff
@@ -14,7 +14,7 @@ scifi_nodes:slope_vent 120 120 120
 scifi_nodes:white2 240 240 240
 ```
 
-Default colors, see: [colors.txt](../server/static/colors.txt)
+Default colors, see: [colors directory](../public/colors)
```

Comment:
I think this refers to an old implementation, so I just updated this to the directory where colours are stored.